proposals/raster_to_vector.json

```diff
@@ -0,0 +1,56 @@
+{
+    "id": "raster_to_vector",
+    "summary": "Converts single band raster data cube to vector data",
+    "description": "Converts a single band raster data cube into a vector data cube. The bounding polygon of uniform areas of pixels is constructed.",
+    "categories": [
+        "cubes"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "data",
+            "description": "A data cube.",
+            "schema": {
+                "type": "object",
+                "subtype": "datacube"
```

Comment:
This requires at least x and y dimensions, right? Should be added as requirement.